VAMobile/src/screens/PaymentsScreen/PaymentHistory/PaymentHistoryScreen.tsx

```diff
@@ -96,36 +101,28 @@ function PaymentHistoryScreen({ navigation }: PaymentHistoryScreenProps) {
   }
 
   const getPaymentsData = (): ReactNode => {
-    const noCurrentPayments = !currentPagePayments || isEmpty(currentPagePayments)
-
-    if (noCurrentPayments) {
+    console.log(payments)
```

Comment:
Looks like this was accidentally left here

Reply:
good catch, removed